doc/tools/make_rst.py

```diff
@@ -1638,6 +1652,7 @@ def make_footer() -> str:
         f".. |static| replace:: :abbr:`static ({static_msg})`\n"
         f".. |operator| replace:: :abbr:`operator ({operator_msg})`\n"
         f".. |bitfield| replace:: :abbr:`BitField ({bitfield_msg})`\n"
```

Comment:
Makes you really think if BitField should have been called BitFlags (on the outside) all along. "field" for these kinds of properties is rarely ever used. Usually they're referred to as "masks", or "flags".
Question for another time, of course.

Reply:
This is the name of the core class. If we ever add such a type to GDScript, it would probably make sense to name it `BitField[T]` for consistency.